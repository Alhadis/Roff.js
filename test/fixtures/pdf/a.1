.ds LQ `
.ds RQ '
.ps 32p
\v'3m'
ABC \*(LQ123\*(RQ XYZ
