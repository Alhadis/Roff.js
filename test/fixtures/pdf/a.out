x T pdf
x res 72000 1 1
x init
p1
V12000
H72000
DFd
wx font 5 TR
f5
s32000
h8000
v96000
md
tABC
wh8000
t`123'
wh8000
tXYZ
n12000 0
x trailer
V792000
x stop
